doc/source/tokudb/tokudb_variables.rst

```diff
@@ -911,6 +921,30 @@ output:
 Defines direction to be used to perform table scan to check for empty tables
 for bulk loader.
 
+.. variable:: tokudb_enable_fast_update
+
+  :cli: Yes
+  :conf: Yes
+  :scope: Global/Session
+  :dyn: Yes
+  :vartype: Boolean
+  :default: Off
+
+Enables the fast updates feature for the UPDATE statement. Fast update involves
```

Comment:
By this description it sounds that it should be just always enabled unconditionally, which is not the case

@georgelorchpercona

Reply:
Correct. This feature poses a risk of index inconsistency in the case of missed key structure validations. Perhaps in some future major version it can be enabled by default. Since it changes behavior and does pose this risk, it is implemented as an optional feature and is disabled by default, thus presenting no change in functionality nor presenting no new risk by default.